TLSharp.Tests/TLSharpTests.cs

```diff
@@ -178,7 +177,7 @@ public virtual async Task SendMessageTest()
 
             if (user == null)
             {
-                throw new System.Exception("Number was not found in Contacts List of user: " + NumberToSendMessage);
+                throw new Exception("Number was not found in Contacts List of user: " + NumberToSendMessage);
```

Comment:
no need for this cosmetic change, please undo (or separate this change into a different commit or PR)

Reply:
will remove but no need for it as Using statement is declared

Reply:
I know, but if we were to add these improvements to any PR, they would become gigantic :) and doing this breaks git-blame